functions/postSurveyInit.ts

```diff
@@ -89,6 +94,27 @@ const triggerPostSurveyFlow = async (context: Context<EnvVars>, channelSid: stri
   return messageResult;
 };
 
+const getTriggerMessage = (event: Body): string => {
+  // Try to retrieve the triggerMessage for the approapriate language (if any)
+  const { taskLanguage } = event;
+  if (taskLanguage) {
+    try {
+      const translation = JSON.parse(
+        Runtime.getAssets()[`/translations/${taskLanguage}/postSurveyMessages.json`].open(),
+      );
+
+      if (!translation.triggerMessage)
+        console.error(`triggerMessage key is missing in translation for ${taskLanguage}`);
```

Comment:
Note that a few of this in production might trigger a page. Should I change to `warn` rather than `error` of is better to raise a lousy alarm if this is missing? (it will indicate some kind of missconfiguration taking place, but maybe not something to page someone).